lib/import-export-visitor.js

```diff
@@ -70,13 +70,14 @@ module.exports = class ImportExportVisitor extends Visitor {
       }
 
       bodyInfo.body = null;
-      bodyInfo.parent = null;
       bodyInfo.hoistedExportsMap = null;
       bodyInfo.hoistedExportsString = null;
       bodyInfo.hoistedImportsString = null;
       bodyInfo.insertCharIndex = null;
       bodyInfo.insertNodeIndex = null;
       bodyInfo.needToAddUseStrictDirective = null;
+      bodyInfo.parent._bodyInfoByName = null;
```

Comment:
Things can actually be simplified since the bodyInfo is discarded when te bodyInfos array is cleared the nulling isn't needed and only the delete remains.